students/bedkowska_julita/lesson_01_basics/sum_of_digits.py

```diff
@@ -0,0 +1,9 @@
+num = int(input("Input a three-digit number: "))
+
+
+def sum(a, b, c):
```

Comment:
sum is built in function, so You redefined it: 
>>> z = [1,2,3]
>>> sum(z)
6
>>> def sum():
...     pass
...
>>> sum(z)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: sum() takes no arguments (1 given)
>>>